resources/lang/en/events/relationship_events/normal_interactions/trust/increase.json

```
@@ -60,7 +60,7 @@
 			"m_c thinks about how r_c is always reliable.",
 			"m_c notices how r_c is being helpful around camp.",
 			"m_c has faith in r_c.",
-			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/poss}s.",
+			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/poss}.",
```

```suggestion
			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/inposs}.",
```
See comment on #3555 . In future, please just highlight that an issue exists in dev and allow us to merge across fixes from 12.0 so that we don't have to read things twice...! <3